cfg/aws/credentials.go

```diff
@@ -88,6 +88,12 @@ func getSession(config *aws.Config) *session.Session {
 		}
 	}
 	log.Printf("D! Successfully created credential sessions\n")
+	cred, err := ses.Config.Credentials.Get()
+	if err != nil {
+		log.Printf("E! Failed to get credential from session: %v", err)
```

Comment:
Not sure about this but should we log this message 
```
log.Printf("E! Failed to get credential from session: %v", err)
```
since if we are not able to get credentials, the log at SDK level will log a better message for us to identify the problem (so its sound duplication logs to me at this level)

Reply:
For example:  if my user does not have enough policy to create dynamo table, it will log this error message
```
Error: operation error DynamoDB: DescribeTable, https response error StatusCode: 400, RequestID: 76GE23N13FTV01B75KIAID8LF7VV4KQNSO5AEMVJF66Q9ASUAAJG, api error AccessDeniedException: User: arn:aws:sts::***:assumed-role/cwagent-integ-test-task-role-b87ce39bcdfebab3/i-0eb654fdefce6012b is not authorized to perform: dynamodb:DescribeTable on resource: arn:aws:dynamodb:us-west-2:***:table/CWAPerformanceMetrics because no identity-based policy allows the dynamodb:DescribeTable action
```
So should we duplicate the failing message here? We could fail fast  but I don't see any value of logging (maybe I'm wrong)
```
log.Printf("E! Failed to get credential from session: %v", err)
```

Reply:
I don't want to rely on logging done by the SDK if we are capturing an error in our code and can control what kind of messaging we emit because of it. It'll be easier to pinpoint an issue if it's a log line that we authored. That's my stance on it